slothy/targets/aarch64/apple_m1_firestorm_experimental.py

```diff
@@ -120,7 +120,7 @@ def get_min_max_objective(slothy):
      vqdmulh_lane,
      vmull, vmlal,
      vsrshr, vushr, vusra, vshl,
-     vand, vbic): ExecutionUnit.V(),
+     vand, vbic, cmge): ExecutionUnit.V(),
```

Comment:
Here you can now refer to `cmge` as `ASimdCompare` instead (assuming all compare instructions have similar performance characteristics).

Reply:
Thanks, I modified the models. However, if we have something like `is_qform_form_of`, then passing the parent class won't work. Do we want to keep it this way or do we want the function returned by is_qform_form_of to also return `True` on the child classes?

Reply:
@dop-amin Good point. I think `is_qform_of` should be able to handle parent classes, yes. I bet the way I did this in `aarch64_neon.py` is not terribly pythonic, but you could use `all_subclass_leaves()` here.